### PR TITLE
raftstore: import macros with the use syntax 

### PR DESCRIPTION
**Pull request by lhy1024**
Signed-off-by: lhy1024 <admin@liudos.us>

### What problem does this PR solve?

Issue Number: supports #9678

### Check List 

Tests 

- Unit test

### Release note 
- No release note

### CODE REVIEW

File: components/raftstore/src/coprocessor/config.rs [hunk 1]
```diff
@@ -5,6 +5,9 @@ use crate::store::SplitCheckTask;
 
 use configuration::{ConfigChange, ConfigManager, Configuration};
 use engine_traits::{config as engine_config, PerfLevel};
+use serde::*;
```
Comment (JmPotato) [inline components/raftstore/src/coprocessor/config.rs:8]: The reason we import macros with the `use` syntax is that we want to make it easy to find the definition and source of imported macros. IMO, using `::*` will go against the original intention why we want to clean up the `extern crate xxx`.

**Review by JmPotato — COMMENTED** [review]
LGTM

**Comment by JmPotato** [comment]
@lhy1024 Please fix the lints!

**Comment by lhy1024** [comment]
@JmPotato

**Review by JmPotato — APPROVED** [review]
LGTM

**Review by HunDunDM — APPROVED** [review]

**Pull request merged**